SofaKernel/extlibs/newmat/CMakeLists.txt

```diff
@@ -42,6 +42,9 @@ add_library(${PROJECT_NAME} STATIC ${HEADER_FILES} ${SOURCE_FILES})
 target_include_directories(${PROJECT_NAME} PUBLIC "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>")
 target_include_directories(${PROJECT_NAME} PUBLIC "$<INSTALL_INTERFACE:include>")
 target_compile_definitions(${PROJECT_NAME} PRIVATE "use_namespace")
+if(SOFA_FLOAT)
+    target_compile_definitions(${PROJECT_NAME} PRIVATE "SOFA_FLOAT")
+endif()
```

Comment:
I do not understand why this is necessary?
Is a dependency to a sofa component missing?